.buildkite/pipeline.yml

```diff
@@ -9,3 +9,12 @@ steps:
     label: crossref-verify itself (import as if done externally)
   - command: nix run -f $(nix eval -f nix/sources.nix --raw nixpkgs) reuse -c reuse lint
     label: reuse lint
+  - command: nix-build release -o result-release
+    label: "Build the release"
+    artifact_paths:
+      - "result-release/bin/*"
+  - command:
+      - GITHUB_TOKEN=$(cat ~/niv-bot-token) NIX_PATH=nixpkgs=channel:nixos-unstable nix-shell -p git gitAndTools.hub --run "curl https://raw.githubusercontent.com/serokell/scratch/587aded2cd1bb624f85b49727272616fdc49f4fc/scripts/release-binary.sh | bash"
+    label: Create a release
```

Comment:
Might be better to call it a "pre-release" here.

Reply:
Fixed.